model/versioning/VersionableExtension.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * Minimum level extra fields required by extensions that are versonable
+ */
+interface VersionableExtension {
```

Comment:
Ah, I can add a line item. It's pretty minor, and I don't think anyone is really using this feature. I half felt it could be removed.